app/services/section/preview_service.rb

```diff
@@ -7,6 +7,14 @@ def initialize(user:, manual_id:, section_uuid:, attributes:)
   end
 
   def call
+    manual = Manual.find(manual_id, user)
+    section = if section_uuid
+                manual.sections.find { |sec|
+                  sec.uuid == section_uuid
+                }
+              else
+                manual.build_section(attributes)
+              end
```

Comment:
Minor: I know Rubocop required it, but the hanging indent makes my eyes bleed. I think you could fix that by extracting a `Manual#find_section` method and converting back to a ternary expression; or even extract the whole thing into `Manual#find_or_build_section`.